src/main/java/com/raizetech7/demo/NameController.java

```diff
@@ -5,23 +5,22 @@
 import org.springframework.web.util.UriComponentsBuilder;
 
 import java.net.URI;
-import java.util.List;
+//import java.util.List;
 import java.util.Map;
 
-import static java.lang.constant.ClassDesc.of;
+//import static java.lang.constant.ClassDesc.of;
 
 @RestController
 public class NameController{
 
-  @GetMapping("/names/{id}")
-  public Name getNames(@PathVariable int id){
+//クエリ文字列を受け取れるように変更
+  @GetMapping("/names")
+  public Name getNames(@RequestParam int id){
 
     return new Name(id,"masami","teramoto");
```

Comment:
fotmatterの設定はいれておきたいところ。
```suggestion
    return new Name(id, "masami", "teramoto");
```

Reply:
action on save 設定ありました。Reformat codeにチェックつけてます。
<img width="1658" alt="スクリーンショット 2023-08-31 14 19 33" src="https://github.com/masami-tr/RaiseTech7/assets/133315049/954dc611-63b2-4346-9ab8-0cce01c6f442">